tools/zos/modifysidedeck.sh

```diff
@@ -12,19 +12,22 @@ originalsidedeck=$1
 outputsidedeck=$2
 newdllname=$3
 
+SCRIPT_DIR=$(dirname $0)
 ID=`date +%C%y%m%d_%H%M%S`
-TMP="/tmp/sidedeck-$(basename $0).$ID.tmp"
-TMP2="/tmp/sidedeck-$(basename $0).$ID.tmp.2"
+TMP="/tmp/sidedeck-$(basename "$0").$ID.tmp"
+TMP2="/tmp/sidedeck-$(basename "$0").$ID.tmp.2"
```

Comment:
Double quotes inside double quotes? I think the double quotes are instead needed for the SCRIPT_DIR and ID assignments.

Reply:
it was one of linter's complain, and within `$()` the outside quote does not apply